muxpi/set-up-image.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+set -e
+
+#If you want to run the neptune image, run `./set-up-image.sh neptune`
```

Comment:
You could have this as a "usage" comment. Also include some description of the purpose of the script. Add it above `set -e`.
```suggestion
# <fancy description of the purpose of this script>
# Usage: 
# $ ./set-up-image.sh [neptune]
# The default image to use is the minimal image. If you want to run the neptune image
# use the option 'neptune'.
```